app/HMS/Facades/HmsHelper.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace HMS\Facades;
+
+use Illuminate\Support\Facades\Facade;
+
+/**
+ * @see \HMS\Helpers\HmsHelper
+ */
+class HmsHelper extends Facade
```

Comment:
A less generic name could be more useful? Otherwise this'll just end up a dumping ground for all the odd shite that we can't quickly think of a name for.